packaging/ios/build.sh

```diff
@@ -179,16 +178,16 @@ echo next building for $ARCH
 		perl -pi -e 's/testrecurse\$\(EXEEXT\)//' Makefile
 		make
 		make install
+
 		popd
 	fi
 
-	if [ ! -e libxslt-${LIBXSLT_VERSION}.tar.gz ] ; then
-		# wget ftp://xmlsoft.org/libxml2/libxslt-${LIBXSLT_VERSION}.tar.gz
-		wget http://api.github.com/repos/GNOME/libxslt/tarball/v${LIBXSLT_VERSION} -O libxslt-${LIBXSLT_VERSION}.tar.gz
+	if [ ! -e libxslt-${LIBXSLT_VERSION}.tar.xz ] ; then
+		curl -O https://git.gnome.org/browse/libxslt/snapshot/libxslt-${LIBXSLT_VERSION}.tar.xz
```

Comment:
git.gnome might work - do they have libxml as well?
Or... check it out from git as well